Source/WarpX.H

```diff
@@ -665,7 +665,7 @@ public:
      */
     void Hybrid_QED_Push (int lev, PatchType patch_type, amrex::Real dt);
 
-    static amrex::Real quantum_xi_c2;
+    amrex::Real quantum_xi_c2;
```

Comment:
Name this `m_...`?
```suggestion
    amrex::Real m_quantum_xi_c2;
```

Reply:
ah, yes, that's a good point... I will rename the variable everywhere.

Reply:
Done